### PR TITLE
Update ETL pipeline readme 

### PR DESCRIPTION
**Pull request by ayahusseini**
- updated the ETL pipeline with setup instructions, overview, and some details on implementation

### CODE REVIEW

**Comment by ayahusseini** [comment]
@megansam5 , I put in a to-do box for where you could include more detail on OpenAI stuff if needed/if I need to be corrected

File: news_sentiment_analyser/README.md [hunk 1]
```diff
@@ -1,23 +1,71 @@
...
+------------------------------------------------
+clean_content.py                17      1    94%
+database_functions.py           39      7    82%
+extract_s3.py                   37      1    97%
+load_rds.py                     40      3    92%
+openai_topics.py                55      6    89%
+sentiment_analysis.py           18      1    94%
+------------------------------------------------
+TOTAL                          548     19    97%
+```
```
Comment (indiah444) [inline news_sentiment_analyser/README.md:65]: Fantastic idea to put the coverage information in the README!

File: news_sentiment_analyser/README.md [hunk 2]
```diff
@@ -1,23 +1,71 @@
...
+
+## Methodology 
 
+Sentiment analysis is performed using (VADER)[https://ojs.aaai.org/index.php/ICWSM/article/view/14550]. VADER is a rule-based sentiment analyser which can map the "intensity" and nature of emotions to a score. Passed to the database is a compound score for the `title` and `content`, normalised to lie in the range $[-1,1]$. As for topic labelling, we begin with a pre-set list. Articles are mapped to 0, 1 or more topics, depending on their title. This is handled using `gpt-4o-mini` OpenAI model.
```
Comment (indiah444) [inline news_sentiment_analyser/README.md:70]: Good idea to add in this extra info and link to relevant websites.

**Review by indiah444 — APPROVED** [review]
Great improvement to the README, very clear with essential info :)

**Pull request merged**